### PR TITLE
adding html-css-popup task

### PR DESCRIPTION
**Pull request by YarHadi**
### HTML-CSS-practice-popup_task

[Demo](https://yarhadi.github.io/hooli-popup/) |
[Code base](https://github.com/YarHadi/hooli-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by kasionio** [comment]
Nice job! But I can't reach the popup from the keyboard
image

**Comment by YarHadi** [comment]
Hello, fixed reaching the popup from the keyboard.
I'm waiting for new comments)))

**Review by kasionio — CHANGES_REQUESTED** [review]
Greta job! Please take a look at the comments below. And make the width of all items containers the same (screenshot below)
image

**Comment by YarHadi** [comment]
Hi. All comments resolved, please take another look.
Thanks you)

**Comment by kasionio** [comment]
Please click here, this way I can understand that you are done and re-review is needed

File: submissions/YarHadi/html-css-popup/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  display: flex;
+  color: rgb(255, 255, 255);
+  justify-content: center;
+  text-align: center;
+  padding: 16px;
+  text-decoration: none;
+}
+
+.nav-item label:hover {
+  background-color: #111111;
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/css/styles.css:56]: 👀

File: submissions/YarHadi/html-css-popup/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  cursor: pointer;
+}
+
+.nav-item label:focus {
+  background-color: rgba(0, 225, 255, 0.281);
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/css/styles.css:61]: pretty strange decisions of choosing the colors. why do you need alpha with such accuracy?

File: submissions/YarHadi/html-css-popup/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+.nav-img {
+  height: 30px;
+}
+
+.user {
+  width: 30px;
+  object-fit: cover;
+  border-radius: 100%;
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/css/styles.css:71]: 👍

File: submissions/YarHadi/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+              <li class="popup-menu-item">
+                <a class="submenu-link" href="#">
+                  <img src="./img/documents.png" class="popup-img" alt="A piece of paper with the text shown schematically" />
+                  <p>Documents</p>
+                </a>
+              </li>
+            </ul>
+            <input type="checkbox" class="pop-input-ins" id="popup-more-btn" />
+            <label for="popup-more-btn" class="more-btn" id="popup-more-btn-label"> More </label>
+            <hr class="popup-line" />
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/index.html:110]: 👍

File: submissions/YarHadi/html-css-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+              <li class="popup-menu-item">
+                <a class="submenu-link" href="#">
+                  <img src="./img/maps.png" class="popup-img" alt="red pointer on the schematic map" />
+                  <p>Map</p>
+                </a>
+              </li>
+              <li class="popup-menu-item">
+                <a class="submenu-link" href="#">
+                  <img src="./img/files.png" class="popup-img" alt="Yellow folder for papers" />
+                  <p>Files</p>
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/index.html:62]: is `p` tag is needed here?

File: submissions/YarHadi/html-css-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>POPUP</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <ul class="nav-menu">
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/index.html:12]: this is a navigation menu 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav?retiredLocale=uk

File: submissions/YarHadi/html-css-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+              <li class="popup-menu-item">
+                <a class="submenu-link" href="#">
+                  <img src="./img/blogs.png" class="popup-img" alt="
+                  Set of articles on an orange background" />
+                  <p>Blogs</p>
+                </a>
+              </li>
+              <li class="popup-menu-item">
+                <a class="submenu-link" href="#">
+                  <img src="./img/promo.png" class="popup-img" alt="Yellow horn on a blue background" />
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/index.html:145]: you can change `user` alt the same way

**Review by kasionio — CHANGES_REQUESTED** [review]
Good job! Some UI fixes are needed. Padding between container and element should be equal from both sides. Also seems padding from the last elements to the bottom of container is too small.
image

**Comment by kasionio** [comment]
BTW, the mobile view looks really great, good job!

**Comment by YarHadi** [comment]
Hello. All comments resolved.
I hope you like it.
Waiting for new comments))

**Comment by kasionio** [comment]
> Hello. All comments resolved. I hope you like it. Waiting for new comments))

No need to make what I like)) If you don't understand the reason why to do so just ask ;)

File: submissions/YarHadi/html-css-popup/index.html [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+                  Promos
+                </a>
+              </li>
+            </ul>
+          </div>
+        </li>
+        <li class="nav-item">
+          <a href="https://github.com/YarHadi/kottans-frontend" class="navmenu-link" target="_blank">
+            <img src="./img/user-photo.png" class="nav-img user" alt="A handsome young man with an elegant beard" />
```
Comment (kasionio) [inline submissions/YarHadi/html-css-popup/index.html:154]: 😎

File: submissions/YarHadi/html-css-popup/index.html (hunk 6 above)
Comment (kasionio) [reply submissions/YarHadi/html-css-popup/index.html:12]: But no need to remove a header, just add a nav))

Comment (kasionio) [reply submissions/YarHadi/html-css-popup/index.html:12]: In the top of the site we have a header, which contains nav menu on it. Think about it this way

**Review by kasionio — CHANGES_REQUESTED** [review]

File: submissions/YarHadi/html-css-popup/index.html (hunk 6 above)
Comment (YarHadi) [reply submissions/YarHadi/html-css-popup/index.html]: Sorry. My bad. 
Fixed

File: submissions/YarHadi/html-css-popup/index.html (hunk 8 above)
Comment (YarHadi) [reply submissions/YarHadi/html-css-popup/index.html]: 😎

**Review by kasionio — APPROVED** [review]
Good job ;) Lets move forward 🚀
Personal recommendation-  read about relative units as rem and em, nowadays they are widely used

**Comment by YarHadi** [comment]
Thanks for your patience and help.
Also thanks for the recommendation.

Only forward.🚀
To infinity and beyond.

**Pull request merged**